picard/ui/caa_types_selector.py

```diff
@@ -201,7 +205,10 @@ def __init__(
 
         # Add instructions to the dialog box
         instructions = QtWidgets.QLabel()
-        instructions.setText(_("Please select the contents of the image type 'Include' and 'Exclude' lists."))
+        if instructions_top:
+            instructions.setText(_(instructions_top))
+        else:
+            instructions.setText(_("Please select the contents of the image type 'Include' and 'Exclude' lists."))
```

Comment:
Just a minor thing, but instead repeating calls we may just do:

```python
        if instructions_top is None:
           instructions_top = N_("Please select the contents of the image type 'Include' and 'Exclude' lists.")
        instructions.setText(_(instructions_top))
```

Reply:
Great suggestion.  Thanks.